### PR TITLE
[Edge] leaf device to query pipeline app

### PR DESCRIPTION
**Pull request by gichan-jang**
Application that transmits data from leaf device that uses only nns_edge to edge server that uses nnstreamer query pipeline.

Signed-off-by: gichan <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: native/example_inference_offloading/nnstreamer_example_inference_offloading_edge.c [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  loop = g_main_loop_new (NULL, FALSE);
+
+  /* init pipeline */
+  str_pipeline =
+      g_strdup_printf
+      ("tensor_query_serversrc ! video/x-raw,width=224,height=224,format=RGB,framerate=0/1 ! "
+      "tensor_converter ! tensor_filter framework=custom-easy model=infrecne_offloading ! "
+      "tensor_query_serversink");
+
+  NNS_custom_easy_register ("infrecne_offloading", ce_custom_inference_offloading, NULL, &info_in, &info_out);
```
Comment (jaeyun-jung) [inline native/example_inference_offloading/nnstreamer_example_inference_offloading_edge.c:68]: typo : infrecne_offloading > inefrecne_offloading at line 65, 68, and 89

Comment (gichan-jang) [reply native/example_inference_offloading/nnstreamer_example_inference_offloading_edge.c]: Thanks, Updated.

File: Tizen.platform/Tizen_IoT_inference_offloading/nnstreamer_example_inference_offloading_leaf_rt.c [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    switch (opt) {
+      case 'h':
+        free (opt_data->host);
+        opt_data->host = strdup (optarg);
+        break;
+      case 'p':
+        opt_data->port = (uint) strtoll (optarg, NULL, 10);
+        break;
+      case 't':
+				opt_data->topic = strdup (optarg);
```
Comment (jaeyun-jung) [inline Tizen.platform/Tizen_IoT_inference_offloading/nnstreamer_example_inference_offloading_leaf_rt.c:148]: plz fix indent.

File: Tizen.platform/Tizen_IoT_inference_offloading/meson.build [hunk 3]
```diff
@@ -0,0 +1,46 @@
+project('nnstreamer-example', 'c', 'cpp',
```
Comment (jaeyun-jung) [inline Tizen.platform/Tizen_IoT_inference_offloading/meson.build:1]: Q: should we add new project for this example?
plz consider adding new package in nnstreamer-example.spec file.

File: Tizen.platform/Tizen_IoT_inference_offloading/nnstreamer_example_inference_offloading_leaf_rt.c (hunk 2 above)
Comment (gichan-jang) [reply Tizen.platform/Tizen_IoT_inference_offloading/nnstreamer_example_inference_offloading_leaf_rt.c]: Updated. Thanks.

File: Tizen.platform/Tizen_IoT_inference_offloading/meson.build (hunk 3 above)
Comment (gichan-jang) [reply Tizen.platform/Tizen_IoT_inference_offloading/meson.build:1]: I  separated the project to reduce build time. I will modify it with other project under Tizen.platform in the next PR.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**